<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_play</class>
 <widget class="QDialog" name="window_play">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>618</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="p2s">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>120</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>35</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Game">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 211, 211);
font: 10pt &quot;Siemens AD Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Games played: 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendnameButton">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
       border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: 9pt &quot;Fixedsys&quot;;
		

}

QPushButton:enabled {
		background-color: #484848;
		color: rgb(211, 211, 211);
}

QPushButton:pressed {
		
		background-color: black;
		image: url(:/Img/back.png);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #696c9d;

		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}

</string>
   </property>
   <property name="text">
    <string>Send name</string>
   </property>
  </widget>
  <widget class="QLabel" name="me_name_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>130</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Siemens AD Sans</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Siemens AD Sans&quot;;
color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Doyouhavename_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 211, 211);
font: 10pt &quot;Siemens AD Sans&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>130</y>
     <width>111</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Siemens AD Sans</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Siemens AD Sans&quot;;
color: rgb(211, 211, 211);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="write_name_Edit">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1s">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>120</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>35</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
       border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		image: url(:/Img/save.png);
	
}

QPushButton:enabled {
		

	
	background-color: rgb(0, 33, 61);
	background-repeat: no-repeat;
    background-position: center;
		color: white;
}

QPushButton:pressed {
		
		background-color: black;
		image: url(:/Img/save.png);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #696c9d;

		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Backbutton3">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>530</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
       border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		image: url(:/Img/back.png);

}

QPushButton:enabled {
		
		background-color: rgb(0, 33, 61);
	
	color: rgb(170, 0, 0);
		color: white;
}

QPushButton:pressed {
		
		background-color: white;
		image: url(:/Img/back.png);
		color: #ffffff;
}


QPushButton:hover:!pressed {
		background-color: #696c9d;

		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #ffffff;

		color: #ffffff;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>590</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(121, 120, 117);

font: 10pt &quot;Siemens AD Sans&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="turnLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>79</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Siemens Slab Black</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Fixedsys&quot;;
color: rgb(211, 211, 211);
font: 18pt &quot;Siemens Slab Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>120</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipLine">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 11px;</string>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLine">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>230</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 11px;</string>
   </property>
   <property name="text">
    <string>2323</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>195</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Fixedsys&quot;;
color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Fixedsys&quot;;
color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
       border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: 9pt &quot;Fixedsys&quot;;
}
QPushButton:enabled {
		
	background-color: rgb(0, 33, 61);
		color: white;
}

QPushButton:pressed {
		
		background-color: black;
		image: url(:/Img/back.png);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #696c9d;

		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rematchButton">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
       border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: 9pt &quot;Fixedsys&quot;;
		

}

QPushButton:enabled {
		
	background-color: rgb(168, 172, 198);
		color: white;
}

QPushButton:pressed {
		
		background-color: black;
		image: url(:/Img/back.png);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #696c9d;

		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}

</string>
   </property>
   <property name="text">
    <string>Rematch</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>224</y>
     <width>10</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 15px;
background-color: rgb(0, 33, 61);


</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>224</y>
     <width>10</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 15px;
background-color: rgb(0, 33, 61);

</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>225</y>
     <width>10</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 15px;
background-color: rgb(0, 33, 61);

</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>244</y>
     <width>230</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 15px;

background-color: rgb(0, 33, 61);

</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>326</y>
     <width>230</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border-radius: 15px;

background-color: rgb(0, 33, 61);

</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>414</y>
     <width>230</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Fixedsys&quot;;
background-color: rgb(0, 33, 61);
border-radius: 15px;


</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>195</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">		border-image: url(:/Img/Space..jpg);
		color: rgb(255, 255, 255);
      	border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		


</string>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Img/white.jfif);
border-image: url(:/Img/Space..jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>140</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Siemens AD Sans&quot;;
color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Img/diagonal.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Img/diagonal_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>230</y>
     <width>70</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>230</y>
     <width>70</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>230</y>
     <width>70</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>14</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Siemens AD Sans&quot;;
color: rgb(211, 211, 211);</string>
   </property>
   <property name="text">
    <string>TIC TAC TOE</string>
   </property>
  </widget>
  <widget class="QLabel" name="frame">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-10</y>
     <width>511</width>
     <height>691</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-origin: content;



	background-repeat: no-repeat;
    background-position: center;


  	border-image: url(:/Img/Space..jpg);
	
    background-repeat: no-repeat;
    background-origin: content;
	color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>241</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>232</y>
     <width>241</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>315</y>
     <width>241</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>70</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>70</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>gridWidget</zorder>
  <zorder>line_6</zorder>
  <zorder>line_3</zorder>
  <zorder>line_2</zorder>
  <zorder>line</zorder>
  <zorder>line_7</zorder>
  <zorder>line_5</zorder>
  <zorder>label_9</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>write_name_Edit</zorder>
  <zorder>label_8</zorder>
  <zorder>connectButton</zorder>
  <zorder>portLine</zorder>
  <zorder>ipLine</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>SendnameButton</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_3</zorder>
  <zorder>p2s</zorder>
  <zorder>p1s</zorder>
  <zorder>label_6</zorder>
  <zorder>me_name_label</zorder>
  <zorder>Doyouhavename_label</zorder>
  <zorder>Game</zorder>
  <zorder>Backbutton3</zorder>
  <zorder>rematchButton</zorder>
  <zorder>Save</zorder>
  <zorder>label_4</zorder>
  <zorder>turnLabel</zorder>
  <zorder>label_15</zorder>
  <zorder>label_14</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
